### DISCUSSION TITLE
MediaElement options based registration

### DISCUSSION THREAD

**Discussion by bijington** [CommunityToolkit/Maui#1846] [General]
We currently have this section in the documentation: https://learn.microsoft.com/en-gb/dotnet/communitytoolkit/maui/views/mediaelement?tabs=windows#bypassing-the-ios-silent-switch

Should we opt for an options based approach when allowing developers to configure the audio playback settings (and possibly more) like this approach: https://github.com/jfversluis/Plugin.Maui.Audio/blob/main/docs/audio-player.md#configure-the-playback-options

**Comment by CliffAgius** [comment]
@ne0rrmatrix @bijington Is this being looked at or can we close this discussion item.

**Reply by ne0rrmatrix** [reply]
It has been discussed and there are ideas about this. I don't have any plans in the near future to address it. But I do plan on looking into this with the whole options class being something I do very much want in media element. Having device specific options would be nice to have. would allow simple registration of things like, disable android service(in case of no we do not want notifications or android service issues), various avplayer options in IOS)
